src/demo-app/progress-fab/progress-fab-demo.scss

```diff
@@ -0,0 +1,26 @@
+.demo-progress-fab {
+
+  .demo-progress-card {
+    padding: 0;
+    margin: 16px;
+
+    .demo-content {
+      padding: 7px;
+
+      .demo-determinate-progress.hide-progress {
+        .md-progress-wrap {
+          md-progress-circle {
+            transition-duration: 0.7s;
+            transition-property: transform, opacity;
+            transition-timing-function: ease-out;
+
+            transform: scale(0.8);
+            opacity: 0;
+          }
+        }
+      }
+
+    }
+  }
```

Comment:
wouldn't that hurt, albeit just a bit, maintainability? In the same way single-line 'if' blocks not surrounded by {} are usually considered kinda rude to the next developer who has to add further statements and is forced to add the braces and reindent